readme.md

```diff
@@ -136,6 +136,14 @@ The attribute should be an image URL:
 
 Read more about the `media` parameter in the [in the Pinterest documentation](https://developers.pinterest.com/docs/widgets/pin-it/#source-settings).
 
+### Telegram
+
+You can set `data-text` attribute to set a text of the message.
+
+```html
+<div class="telegram" data-text="Hello, world!">Send</div>
```

Comment:
“Hello, world!” sounds too example-ish to me. Can we use something more real here? The button is supposed to be used for sharing. What text can a real website append next to the link that is general enough but still real?